webpack.config.js

```diff
@@ -250,6 +247,9 @@ var config = {
             superagent: 'node_modules/superagent/lib/client',
         },
         extensions: ['.ts', '.tsx', '.js', '.jsx'],
+        fallback: {
+            stream: require.resolve('stream-browserify'),
```

Comment:
Similar to `crypto`, we need to import `stream` ourselves as well. Since we don't use it directly (it's used by `crypto-browserify`), we have to tell Webpack to alias it instead of updating import paths